packages/svelte/src/compiler/phases/3-transform/client/visitors/template.js

```diff
@@ -2630,7 +2630,7 @@ export const template_visitors = {
 				);
 
 				// we need to eagerly evaluate the expression in order to hit any
-				// 'Cannot access x before initialization' errors
+				// 'Cannot access x before initialisation' errors
```

Comment:
```suggestion
				// 'Cannot access x before initialization' errors
```

We should use the actual error message here as well.